assignment1.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Assignemnt1</title>
+    <style>
+
+      table, th, td {
+      border: 1px solid black;
+      border-collapse: collapse;
+      padding: 5px;
+      }
+      
+      th{
+        font-weight: bold;
+        background-color: rgb(195, 195, 195);
+      }
+      .State {
+        background-color: rgb(195, 195, 195);
+      }
+    </style>
+    <table>
```

Comment:
table should be inside body not inside head

Reply:
ok.modified